README.md

```diff
@@ -91,6 +89,10 @@ Global orders are a new type of order for trading on Solana. When resting orders
 cargo build-sbf
 ```
 
+### Open Questions
+- Is tickless a good idea? This inverts time priority since it makes the most recent order able to provide negligible price improvement. This could disrupt behavior near mid and lead to unforeseen patterns.
```

Comment:
I would counter this that it makes it a more competitive venue and purer base primitive. Dont know if it's important to mention

Reply:
Agree that it is purer, but the benefits from it are really mostly marketing.
We will see if it causes issues. I dont think it will, but it is something we should be aware of. There is a possibility that weird effects happen, like lots of frontrunning MEV, or tight quoters backing off since they will not as likely get the good fills.

Just highlighting these open questions because everything else is pretty much a pure positive over existing orderbooks. These are the main tradeoffs to consider.